tests/fe/nedelec_embed_dofs.cc

```diff
@@ -0,0 +1,90 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2017 - 2021 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+// Checks if get_embedding_dofs() correctly maps the local DoF indices
+// of an FE_Nedelec element of sub_degree to an element of sup_degree,
+// where sub_degree <= sup_degree
+
+#include <deal.II/base/quadrature_lib.h>
+
+#include <deal.II/fe/fe_nedelec.h>
+
+#include "interpolate_common.h"
+
+template <unsigned int dim>
```

Comment:
```suggestion
template <int dim>
```
You are right technically. Throughout the library however, we use just `int dim` for template parameters, so I suggest to do so here as well.